tests/FakeItEasy.Specs/ManageEventsSpecs.cs

```diff
@@ -111,5 +111,30 @@ public void ManageAllEvents(IFoo fake, EventHandler barHandler, EventHandler baz
             "And the handler for Baz is called"
                 .x(() => A.CallTo(bazHandler).MustHaveHappened());
         }
+
+        [Scenario]
+        public void MultipleCallsToManageEvent(IFoo fake, EventHandler handler)
+        {
+            "Given a strict fake"
+                .x(() => fake = A.Fake<IFoo>(o => o.Strict()));
+
+            "And an event handler"
+                .x(() => handler = A.Fake<EventHandler>());
+
+            "When the fake is configured to manage the Bar event"
+                .x(() => Manage.Event(nameof(IFoo.Bar)).Of(fake));
+
+            "And the handler is subscribed to the Bar event"
+                .x(() => fake.Bar += handler);
+
+            "And the fake is configured again to manage the Bar event"
+                .x(() => Manage.Event(nameof(IFoo.Bar)).Of(fake));
+
+            "And the Bar event is raised using Raise"
+                .x(() => fake.Bar += Raise.WithEmpty());
+
+            "Then the handler is called"
+                .x(() => A.CallTo(handler).MustHaveHappened());

```

Comment:
Exactly once? Or is there no value in checking that we don't raise twice after managing twice?